src/v2/Apps/Auctions/auctionsRoutes.tsx

```diff
@@ -36,6 +36,7 @@ export const auctionsRoutes = [
       {
         path: "", // represents current auctions aka /auctions/current
         getComponent: () => CurrentAuctions,
+        ignoreScrollBehavior: true,
```

Comment:
Fixes the issue where when clicking the tabs, the page scrolls all the way to the top.